connector/src/main/scala/com/basho/riak/spark/writer/WriteConf.scala

```diff
@@ -31,4 +37,9 @@ object WriteConf {
       writeQuorum = conf.getInt(WriteQuorumProperty, DefaultWriteQuorum)
     )
   }
+  
+  def fromOptions(options: Map[String, String], conf: SparkConf): WriteConf = {
```

Comment:
Such method is not possible since Scala has limitation on number of methods with default values and there already is a WriteConf.apply with defaults.